components/places/src/import/fennec.rs

```diff
@@ -1,9 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file I forgot to remove when removing the fennec migration code, i removed all references to it though!